dist/openshift/cincinnati-deployment.yaml

```
@@ -410,16 +464,24 @@ parameters:
 
       [[plugin_settings]]
       name = "edge-add-remove"
+  - name: PE_UPSTREAM_URL
+    value: "http://cincinnati-graph-builder.cincinnati.svc.cluster.local:8080${GB_PATH_PREFIX}/graph"
```

I am not sure whether we can assume that the application will be deployed in the `cincinnati` namespace. For example, we use the `cincinnati-production` namespace in production. This template is used in https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/cincinnati/cicd/ci-int/saas.yaml, which ends up deploying Cincinnati in various namespaces.

---

We might use just the service name given the pods are deployed in the same namespace. For example:

```suggestion
    value: "http://cincinnati-graph-builder:8080${GB_PATH_PREFIX}/graph"
```